DOL.WHD.Section14c.Web/webpack.config.js

```diff
@@ -82,6 +82,14 @@ module.exports = {
     new webpack.ProvidePlugin({
       $: 'jquery',
       jQuery: 'jquery'
-    })
+    }),
+    new webpack.DefinePlugin({
+      'window.__env': {
+        api_url: JSON.stringify(process.env.api_url) || JSON.stringify('https://localhost:44399'),
+        reCaptchaSiteKey: JSON.stringify('6LeqeggUAAAAALC5zT4OHbDJk9gHNT0GGZbJMOnG'),
+        requireHttps: JSON.stringify(true),
+        tokenCookieDurationMinutes: JSON.stringify(20160)
+      }
+    }),
   ]
```

Comment:
_*Love*_ it.  In the future, I'd love to see the captcha key, HTTPS flag, and cookie duration also turned into env vars.  Maybe I should make a little chore story for that.  This is a great step, in any case.  😄